docs/products/kafka/karapace/howto/manage-schema-registry-authorization.rst

```diff
@@ -0,0 +1,33 @@
+Manage Karapace schema registry authorization
+=============================================
+
+Karapace schema registry authorization allows you to authenticate the user, to control access to individual `Karapace schema registry REST API endpoints <https://github.com/aiven/karapace>`_, and to filter the content the endpoints return.
```

Comment:
Don't we have a better REST API endpoint? I would suggest to create a reference to the API endpoints section and use the `:ref:` to link to that?